src/model.ts

```diff
@@ -420,7 +421,7 @@ export default class MediaPlayerObject {
           return this.handleSoundtouch(e, 'REMOVE_ZONE_SLAVE', entity);
         case PLATFORM.SQUEEZEBOX:
           return this.callService(e, 'unsync', options, PLATFORM.SQUEEZEBOX);
-        case PLATFORM.MEDIAPLAYER:
+        case PLATFORM.MEDIAPLAYER || PLATFORM.SONOS:
```

Comment:
Same as previous comment

```suggestion
        case PLATFORM.MEDIAPLAYER:
        case PLATFORM.SONOS:
```